js/main.js

```diff
@@ -87,3 +116,20 @@ function viewEvent(event) {
 
 button.addEventListener('click', viewEvent);
 nav.addEventListener('click', viewEvent);
+
+theUL.addEventListener('click', function (event) {
+  if (event.target.matches('.icon')) {
+    viewSwitch('entry-form');
+    var parentLi = event.target.closest('li');
+    var parentIdJSON = parentLi.getAttribute('data-entry-id');
+    var parentId = JSON.parse(parentIdJSON);
```

Comment:
You shouldn't use `JSON.parse` to convert to a number, you should just use the `parseInt()` function.